qabelbox/src/main/java/de/qabel/qabelbox/contacts/dto/ContactDto.kt

```diff
@@ -3,4 +3,8 @@ package de.qabel.qabelbox.contacts.dto
 import de.qabel.core.config.Contact
 import de.qabel.core.config.Identity
 
-data class ContactDto(val contact : Contact, val identities : List<Identity>)
+data class ContactDto(val contact: Contact, val identities: List<Identity>) {
+
+    var active: Boolean = true
```

Comment:
Is the var really needed? Creating a new ContactDto with a changed active value which is provided in the constructor (with default value=true) would be cleaner.